example/transform/huggingface_model_json.ipynb

```diff
@@ -0,0 +1,391 @@
+      "  warnings.warn(\n",
+      "100%|██████████| 7/7 [04:39<00:00, 39.94s/it]\n"
+     ]
+    }
+   ],
+   "source": [
+    "output = client.run(input_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Process the output\n",
+    "\n",
+    "Let's take a look of the generated output, which is already a list of JSON"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'context': 'We believe our success depends upon our capabilities in areas '\n",
+      "            'such as design, research and development, production and '\n",
+      "            'marketing and is supported and protected by our intellectual '\n",
+      "            'property rights, such as trademarks, utility and design patents, '\n",
+      "            'copyrights, and trade secrets, among others. We have followed a '\n",
+      "            'policy of applying for and registering intellectual property '\n",
+      "            'rights in the United States and select foreign countries on '\n",
+      "            'trademarks, inventions, innovations and designs that we deem '\n",
+      "            'valuable. W e also continue to vigorously protect our '\n",
+      "            'intellectual property, including trademarks, patents and trade '\n",
+      "            'secrets against third-party infringement and misappropriation.',\n",
+      " 'question': 'Which intellectual property rights does the company actively '\n",
+      "             'pursue and protect?',\n",
+      " 'answer': 'The company applies for and registers trademarks, utilizes patents '\n",
+      "           'for inventions and innovations, holds copyrights, and guards trade '\n",
+      "           'secrets.'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "from pprint import pprint\n",
+    "\n",
+    "result = output[0]['output'][0]['response'][0] ## we only postprocess the first output\n",
+    "\n",
+    "pprint(result, sort_dicts=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Congrats! Your question answers from the given knowledge context are generated!\n",
+    "\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n",
+    "## Appendix\n",
+    "\n",
+    "We benchmarked to see the optimal `batch_size` for the `TransformQAHuggingFaceJsonFormatConfig` flow. The answer is \"It depends on your data token length, your GPU memory, your LLM size, etc.\" In the following experiment, we use a GPU with 24G memory and a quantized LLM (2G). We still use the above 400 raw data strings `raw_context_input_400`.\n",
```

Comment:
You might also want to include what compute resource you are using such as AWS instance type.